input/fsh/examples/cinc/CommunicationRequestEmailExample.fsh

```diff
@@ -0,0 +1,74 @@
+* identifier[2].value = "#jb8d66-1390-42c7-89a6-a9a75d65d344"
+* identifier[2].system = "https://api.messaging.digital.health.nz/MessageID"
+* identifier[3].use = #secondary
+* identifier[3].value = "#kb8d66-1390-42c7-89a6-a9a75d65d333"
+* identifier[3].system = "https://api.messaging.digital.health.nz/ProviderMessageID"
+* identifier[4].use = #secondary
+* identifier[4].value = "#lb8d66-1390-42c7-89a6-a9a75d65d333"
+* identifier[4].system = "https://api.messaging.digital.health.nz/ConsumerID"
+* status = #unknown
+* medium.coding.code = #MAILWRIT
+* medium.coding.system = "http://terminology.hl7.org/CodeSystem/v3-ParticipationMode"
+* occurrenceDateTime = "2024-08-20T14:30:00Z"
+* authoredOn = "2024-08-20T14:30:00Z"
+* recipient.type = "Practitioner"
+* recipient.identifier.use = #official
+* recipient.identifier.system = "https://standards.digital.health.nz/ns/hpi-person-id"
+* recipient.identifier.value = "99ZZFX"
+* subject.identifier.use = #official
+* subject.identifier.system = "https://standards.digital.health.nz/ns/nhi-id"
+* subject.identifier.value = "ZXP7823"
+* subject.display = "Carey Carrington"
+* sender = Reference(BSAService)
+//Email Body
+* payload[0].contentAttachment.contentType = #application/pdf
+* payload[=].contentAttachment.language = #en-nz
+* payload[=].contentAttachment.id = "attachment-body"
+* payload[=].contentAttachment.title = "Appointment Reminder" //Email Subject
+* payload[=].contentAttachment.creation = "2023-08-01T09:35:00+11:00"
+* payload[=].contentAttachment.data = "IyBQYXRpZW50IEVuY291bnRlciBOb3RlcwoKIyMgMjAyMy0wOC0wMgoKLSBEaXNjdXNzZWQgY29uZGl0aW9uCi0gT3RhaW5lZCBjb25zZW50IAotIGV0Yy4="
+
+//Attachment1
+* payload[+].contentAttachment.contentType = #application/pdf
+* payload[=].contentAttachment.id = "attcahment-file1"
+* payload[=].contentAttachment.title = "File1.pdf"
+* payload[=].contentAttachment.creation = "2023-08-01T09:35:00+11:00"
+* payload[=].contentAttachment.data = "IyBQYXRpZW50IEVuY291bnRlciBOb3RlcwoKIyMgMjAyMy0wOC0wMgoKLSBEaXNjdXNzZWQgY29uZGl0aW9uCi0gT3RhaW5lZCBjb25zZW50IAotIGV0Yy4="
+//Attachment2
+* payload[+].contentAttachment.contentType = #application/pdf
+* payload[=].contentAttachment.id = "attcahment-file2"
+* payload[=].contentAttachment.title = "File2.pdf"
+* payload[=].contentAttachment.creation = "2023-08-01T09:35:00+11:00"
+* payload[=].contentAttachment.data = "IyBQYXRpZW50IEVuY291bnRlciBOb3RlcwoKIyMgMjAyMy0wOC0wMgoKLSBEaXNjdXNzZWQgY29uZGl0aW9uCi0gT3RhaW5lZCBjb25zZW50IAotIGV0Yy4="
+//Template
+* payload[+].contentReference.id = "attachment-template"
+* payload[=].contentReference.display = "Vaccination Appointment Reminder"
+* payload[=].contentReference = Reference(DocumentReference/T1238)
+//* payload[=].contentReference.type = "DocumentReference"
+//* payload[=].contentReference.display = "T1238"
+
+//Template Parameter
+* payload[+].contentAttachment.contentType = #application/json
+* payload[=].contentAttachment.id = "attachment-Parameter"
```

Comment:
if this is intended to be a "magic" id name which will be used by a consuming system to identify the parameters used to hydrate the template, it should be consistent with the other magic ID names -- should probably be "attachment-parameters". However, the DocumentReference calls its field "template-variables" so again consistency should be ensured